src/Home.js

```diff
@@ -2,16 +2,27 @@ import React from 'react';
 import ModulePlan from './ModulePlan';
 import PlanningSection from './PlanningSection';
 import moduleplan from './moduleplan.json';
+import users from './users.json';
 
 class Home extends React.Component {
   render() {
+    var userModules = users.students[0].tracked_modules;
     var modules = moduleplan.degree_course.modules;
     var maxSemester = 6;
     var semesters = [1,2,3,4,5,6].map(function(semester) {
```

Comment:
Is the variable name `semesters` still accurate? Like does the name describe the data of the variable well?